changelog.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## unreleased
+
+* [3332](https://github.com/zeta-chain/node/pull/3332) - implement orchestrator V2. Move BTC observer-signer to V2
```

Comment:
shouldn't this be under a feature header?

Reply:
Yes

Reply:
Actually it's refactor